elastic/kibana
# [Fleet] Enabled cert validation on fleet server health check

## Summary

Certificate validation is disabled for the request to `/api/status` on Fleet Server within the `POST /health_check` endpoint. Even though this is an internal request, it has been flagged as a security concern.

The `POST /health_check` endpoint is only accessed from the [Fleet Debugger](https://github.com/elastic/kibana/blob/main/x-pack/plugins/fleet/dev_docs/fleet_debugger.md) at `app/fleet/_debug`, so this a low risk impact change.

### How to test

Run a Fleet Server and load the Fleet Debugger at `app/fleet/_debug`. The Fleet Server health check should work normally.
